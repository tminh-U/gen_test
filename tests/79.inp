13
13 2 1 18 3 17 5 13 15 2 11 16 2 
10
4 12
11 12
12 13
6 9
11 13
1 6
5 8
9 11
7 10
4 4
