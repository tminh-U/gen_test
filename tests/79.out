47
4
3
9
5
18
10
5
9
2
